src/desktop.md

```diff
@@ -494,7 +494,7 @@ see the following resources:
 [Developing packages and plugins]: /docs/development/packages-and-plugins/developing-packages
 [Federated Plugin proposal]: /go/federated-plugins
 [Federated plugins]: /docs/development/packages-and-plugins/developing-packages#federated-plugins
-[How to write a Flutter web plugin, part 2]: https://medium.com/flutter/how-to-write-a-flutter-web-plugin-part-2-afdddb69ece6
+[How to write a Flutter web plugin, part 2]: {{site.medium}}/flutter/how-to-write-a-flutter-web-plugin-part-2-afdddb69ece6
```

Comment:
It looks like there's a separate medium-flutter macro that we could/should be using here:

```suggestion
[How to write a Flutter web plugin, part 2]: {{site.medium-flutter}}/how-to-write-a-flutter-web-plugin-part-2-afdddb69ece6
```